script/openqa-clone-custom-git-refspec

```diff
@@ -8,16 +7,16 @@
 #  openqa-clone-custom-git-refspec https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6529 https://openqa.opensuse.org/tests/835060 DESKTOP=textmode
 #
 set -o pipefail
-if [ -n "$GITHUB_TOKEN" ]; then
+if [[ -n "$GITHUB_TOKEN" ]]; then
```

Comment:
Wouldn't you use `[[` here to get rid of the quotation marks? Your document even explains that. Otherwise we could leave it as-is.

Reply:
I mentioned that point already :-)  
For me not a mandatory change.

Reply:
Then what's the goal of the PR? Causing even more confusion rather than using Bash correctly as per documentation?